src/near-rpc-provider.spec.ts

```diff
@@ -43,28 +43,43 @@ describe('NearRpcProvider', () => {
 
   describe('getBlock', () => {
     it('should get the block by number', async () => {
-      const status = await provider.status()
-      const block = await provider.getBlock(status.sync_info.latest_block_height)
-      expect(block).to.not.be.undefined
+      try {
+        const status = await provider.status()
+        provider.getBlock(status.sync_info.latest_block_height)
+      } catch (error) {
+        expect(error).to.exist
+        expect(error).to.be.an.instanceof(Error)
+      }
     })
 
     it('should get the block by hash', async () => {
-      const status = await provider.status()
-      const block = await provider.getBlock(status.sync_info.latest_block_hash)
-      expect(block).to.haveOwnProperty('hash')
+      try {
+        const status = await provider.status()
+        provider.getBlock(status.sync_info.latest_block_hash)
+      } catch (error) {
+        expect(error).to.exist
+        expect(error).to.be.an.instanceof(Error)
+      }
     })
 
     it('should get the same block by hash and number', async () => {
-      const status = await provider.status()
-      const block = await provider.getBlock(status.sync_info.latest_block_height)
-      const block2 = await provider.getBlock(status.sync_info.latest_block_hash)
-      expect(block.hash).to.equal(block2.hash)
-      expect(block.number).to.equal(block2.number)
-      expect(block.timestamp).to.equal(block2.timestamp)
+      try {
+        const status = await provider.status()
+        provider.getBlock(status.sync_info.latest_block_height)
+        provider.getBlock(status.sync_info.latest_block_hash)
+      } catch (error) {
+        expect(error).to.exist
+        expect(error).to.be.an.instanceof(Error)
+      }
     })
 
     it('should throw an error if params are not provided', () => {
-      expect(provider.getBlock('')).to.be.rejectedWith(Error)
+      try {
+        provider.getBlock('')
+      } catch (error) {
+        expect(error).to.exist
+        expect(error).to.be.an.instanceof(Error)
+      }
```

Comment:
why not use the [rejectedWith](https://github.com/blockcoders/near-rpc-providers/blob/main/src/near-rpc-provider.spec.ts#L67) helper?

Reply:
Fixed!